modules/GT_StatesCard.php

```diff
@@ -69,6 +69,35 @@ function stDrawCard($game)
     return $cardType;
   }
 
+  function stEpidemic($game)
+  {
+    $players = GT_DBPlayer::getPlayersInFlight($game, '', $order = 'DESC');
+
+    foreach ($players as $plId => $player) {
+      $plyrBoard = $game->newPlayerBoard($plId);
+      $plyrContent = $game->newPlayerContent($plId);
+      $crewContent = $plyrContent->getContent("crew");
```

Comment:
This is not part of this ticket, but you could start implementing it now - use constants for content types, like Pavel did for cards. If you set up the framework (e.g. defined all the constants) in this PR then we could just start using them as we continue to build the code.

Reply:
I think I will merge this first, and create a new PR, but yes moving the content types to constants makes a lot of sense.